packages/aws-cdk-lib/aws-stepfunctions-tasks/lib/ecs/run-task.ts

```diff
@@ -347,31 +346,11 @@ export class EcsRunTask extends sfn.TaskStateBase implements ec2.IConnectable {
   private makePolicyStatements(): iam.PolicyStatement[] {
     const stack = cdk.Stack.of(this);
 
-    const taskDefinitionFamilyArn = this.getTaskDefinitionFamilyArn();
-    const reduceRunTaskPermissions = cdk.FeatureFlags.of(this).isEnabled(cxapi.ECS_REDUCE_RUN_TASK_PERMISSIONS);
-    let policyStatements = [];
-
-    // https://docs.aws.amazon.com/step-functions/latest/dg/ecs-iam.html
-    if (reduceRunTaskPermissions) {
-      policyStatements.push(
-        new iam.PolicyStatement({
-          actions: ['ecs:RunTask'],
-          resources: [`${taskDefinitionFamilyArn}:*`],
-        }),
-      );
-    } else {
-      policyStatements.push(
-        new iam.PolicyStatement({
-          actions: ['ecs:RunTask'],
-          resources: [
-            taskDefinitionFamilyArn,
-            `${taskDefinitionFamilyArn}:*`,
-          ],
-        }),
-      );
-    }
-
-    policyStatements.push(
+    const policyStatements = [
+      new iam.PolicyStatement({
+        actions: ['ecs:RunTask'],
+        resources: [`${this.getTaskDefinitionFamilyArn()}:*`],
```

Comment:
If I understand correctly, `this.getTaskDefinitionFamilyArn()` strips away the revision number from `this.props.taskDefinition.taskDefinitionArn`. I am curious, would the CDK user be able to specify specific revision of the task definition if they would like to scope this down for strictest security?

Reply:
Yes, that's correct understanding. 

> I am curious, would the CDK user be able to specify specific revision of the task definition if they would like to scope this down for strictest security

Not easily, they could use escape hatch to scope it down further to a specific revision number.